packages/app/pages/Api/common.tsx

```diff
@@ -0,0 +1,12 @@
+export function concatArray(array: JSON[]) {
```

Comment:
Should add method documentation to specify "none" as a possible return value.

Reply:
resolved

Reply:
Thank you for making this separate from any specific page!
:100: